Documentation/markdown/CMakeOptions.md

```diff
@@ -0,0 +1,218 @@
+| DOXYGEN_GENERATE_DOCUMENTATION           | ON      | This option needs to be selected to build documentation.<br>If selected in the CMake GUI, CMake will automatically find <br>the doxygen and dot executables and display the options <br>below.                                                                                      | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/CMakeLists.txt) 				|
+| DOXYGEN_OPTIONAL_INCLUDE_COMMENTED_ONLY  | ON      | If selected, only documentation for explicitly commented entities <br>will be generated. Otherwise, doxygen will assume that all entities <br>are documented, even if there is no documentation available.                                                                          | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake)	|
+| DOXYGEN_OPTIONAL_INCLUDE_INTERNAL        | OFF     | If selected, this will include all comments preceded by "\internal". <br>These are comments intended for developers only. <br>So if the generated documentation is intended for external users, <br>this option should not be selected.                                             | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake)	|
+| DOXYGEN_OPTIONAL_INCLUDE_EARLY_BINDING   | OFF     | If selected (... and already built), this will include the selected <br>IFC schemas in the documentation. Otherwise, the schemas will not be <br>included in the documentation, but this does not influence the <br>selection of IFC schemas (see EARLYBINDING_ tags in CMake GUI). | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake)	|
+| DOXYGEN_OPTIONAL_AUTO_OPEN_DOCUMENTATION | ON      | If selected, the landing page of the documentation will open automatically. <br>If not selected, you will have to find and open the generated documentation <br>manually (see section below on building the documentation).                                                         | [Go to file](https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake)	|
+
+[Go back to overview of project options](#project-options)
+
+### EarlyBinding
+
+| Option                            | Default | Description | Link to file |
+|-----------------------------------|---------|-------------|--------------|
+| EARLYBINDING_WITH_IFC2X3          | OFF     |             |              |
+| EARLYBINDING_WITH_IFC4            | OFF     |             |              |
+| EARLYBINDING_WITH_IFC4X1          | OFF     |             |              |
+| EARLYBINDING_WITH_IFC4X2          | OFF     |             |              |
+| EARLYBINDING_WITH_IFC4X2_BIM4ROAD | OFF     |             |              |
+| EARLYBINDING_WITH_IFC4X2_DRAFT_1  | OFF     |             |              |
+| EARLYBINDING_WITH_IFC4X3_RC1      | OFF     |             |              |
```

Comment:
I think currently this one is default ON.

Reply:
for all of the schemas?

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/111/commits/1b95985f53ef63395a7d46ec868d8e535215b2df